test/libsolidity/ABIDecoderTests.cpp

```diff
@@ -73,69 +73,6 @@ BOOST_AUTO_TEST_CASE(value_types)
 	)
 }
 
-BOOST_AUTO_TEST_CASE(cleanup)
-{
-	string sourceCode = R"(
-		contract C {
-			function f(uint16 a, int16 b, address c, bytes3 d, bool e)
-					public pure returns (uint v, uint w, uint x, uint y, uint z) {
-				assembly { v := a  w := b x := c y := d z := e}
```

Comment:
Where is the V2 version of this?

Reply:
@leonardoalt and me discussed about this. So, as there is a test that forces V2 in CI, does it make sense to have both copies?

Reply:
Yes, because they behave differently. If you put `ABIEncoderV2Only: true`, then it will not run for v2 at all.

Reply:
Ah, sorry, adding it back now